cmd/habitat/procs/proc.go

```diff
@@ -31,9 +34,13 @@ func NewProc(name, cmdPath, dataPath string, errChan chan ProcError) *Proc {
 func (p *Proc) Start() error {
 	cmd := &exec.Cmd{
 		Path: p.CmdPath,
-		Env: []string{
-			fmt.Sprintf("DATA_DIR=%s", p.DataPath),
-		},
+		Args: append([]string{p.CmdPath}, p.Flags...),
+		Env:  p.Env, // require data dir to be passed in from client
+		/*
+			Env: []string{
```

Comment:
Leftover comment, should we remove?

Reply:
should be removed (this looks like its from an earlier commit)